### PR TITLE
TrilinosCouplings: resolve shadow warning

### PR DESCRIPTION
**Pull request by jhux2**
This actually appears to be a bug.  It appears that the variable `ratio` was intended to be modified within a for-loop, and then used afterwards.  However, it was declared within the loop (hence the shadow), and thus never changing the outer declared variable's value.

I also replaced a bunch of tabs with spaces.

@csiefer2 @trilinos/trilinoscouplings 

## Related Issues

* Blocks #11497.

## Stakeholder Feedback
N/A

### CODE REVIEW

**Comment by jhux2** [comment]
Awaiting feedback from @csiefer2.  The intended logic isn't clear.

**Review by csiefer2 — APPROVED** [review]

File: packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp [hunk 1]
```diff
@@ -252,14 +252,13 @@ class MachineLearningStatistics_Hex3D {
...
-	edge_length_2 = distance2(nodeCoord, node3, node4);
-	double ratio = edge_length_1 / edge_length_2;
-	ratio = std::min(ratio, 1/ratio);
+        node1 = elemToNode(i, edge_nodes_1[edge_opposites_1[j]]);
+        node2 = elemToNode(i, edge_nodes_2[edge_opposites_1[j]]);
+        node3 = elemToNode(i, edge_nodes_1[edge_opposites_2[j]]);
+        node4 = elemToNode(i, edge_nodes_2[edge_opposites_2[j]]);
+        edge_length_1 = distance2(nodeCoord, node1, node2);
+        edge_length_2 = distance2(nodeCoord, node3, node4);
+        ratio = std::min(ratio, 1/ratio);
```
Comment (csiefer2) [inline packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp:261]: Yeah, this ain't right.

**Review by csiefer2 — CHANGES_REQUESTED** [review]

File: packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp (hunk 1 above)
Comment (jhux2) [reply packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp:261]: The question is, what did you intend to happen?

Comment (csiefer2) [reply packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp:261]: Good question

Comment (csiefer2) [reply packages/trilinoscouplings/examples/scaling/TrilinosCouplings_Statistics.hpp:261]: Taper: Maximum ratio of lengths derived from opposite edges.

**Review by csiefer2 — APPROVED** [review]

**Pull request merged**